spec/support/helpers.rb

```diff
@@ -1,4 +1,31 @@
 module Helpers
+  def url_for_resource_collection(resource)
```

Comment:
I found these methods quite cumbersome to parse through in my head when reading them due to the number of things each one can do based on the resource passed in. 

IMO I'd find these easier to read/find how to use if they were more explicit and less generic, e.g. `url_for_recruitment_cycle_collection` and `url_for_provider_collection`

Reply:
ah I see further down there's more methods :thinking: okay i can see why having these generic for now works

Reply:
That's a fair point. I've moved them into `private`, for now at least, since they're not really meant for external use. Maybe it'll better explain the intention that way.